stockflux-securities-master/src/components/SecuritiesTable.js

```diff
@@ -78,17 +102,55 @@ const SecuritiesTable = ({ location }) => {
                 <div className="securities-table-cell">{item.symbol}</div>
                 <div className="securities-table-cell">{item.name}</div>
                 <div className="securities-table-cell">
-                  <Link to={`/inputform/${item.securityId}`}>
-                    <button className="securities-table-button">
-                      <span className="material-icons">edit</span>
+                  <ToolTip message="Edit">
+                    <Link to={`/inputform/${item.securityId}`}>
+                      <button className="securities-table-button">
+                        <FaPen size={20} />
+                      </button>
+                    </Link>
+                  </ToolTip>
+                  <ToolTip message="Delete">
+                    <button
+                      className="securities-table-button"
+                      onClick={() => deleteSecurity(item.securityId)}
+                    >
+                      <FaTrashAlt size={20} />
                     </button>
-                  </Link>
-                  <button
-                    className="securities-table-button"
-                    onClick={() => deleteSecurity(item.securityId)}
-                  >
-                    <span className="material-icons">delete</span>
-                  </button>
+                  </ToolTip>
+                  <ToolTip message={item.visible ? "Hide" : "Show"}>
+                    <button
+                      className={`securities-table-button ${!item.visible &&
+                        "greyed-out"}`}
```

Comment:
`greyed-out` is an odd class name - can we think of an intention of the class name to come up with a better one! `disabled` maybe?

Reply:
Well even though the icon represents the security being disabled I wanted a className that could be used for both `disabled` and `visible`. Plus `disabled` is often used to describe buttons are inactive or cannot be clicked whereas these buttons do work. 

What do you think?

Reply:
I've changed the className from `greyed-out` to `feature-off`